### ISSUE TITLE
Combat XP breakdown by source

### ISSUE THREAD

**Issue by SimulatedKnave** [endless-sky/endless-sky#7833]
### Problem Description

Once I found out the Navy are all dedicated humanitarians, I felt guilty about occasionally having to kill them. 

Then I found out all the pirates are 17 year old kids and idly wondered how many teenagers I'd murdered. 

Then I found out combat XP is the combined enemy crew you have killed. 

And at that point, I officially wanted to know EXACTLY how many humanitarians (Navy), selfless frontier defenders (Militia), desperate teenagers (Pirates), innocent civilians (Merchants) I had slaughtered. Sociopathic squirrels and such could be added as they are discovered. 

(To be clear, for me it's almost entirely pirates. But still). 

Obvious implementation (to me) would be as an expansion of the combat XP tooltip - it would have categories (ideally colour coded) showing how many of each group you'd killed. Governments would simply be labeled by which group they'd appear in. 

### Related Issue Links

I can't find one, but would be a little surprised if it hasn't been suggested before. 

### Desired Solution

Break down combat XP by the governments it came from. I know there are more governments than the player actually sees, but this would seem relatively easy to do. 

### Alternative Approaches

I suppose you could try different implementations of the categories (perhaps simply into "hostiles" and "friendlies"). But a running 'how much of which demographic of the galaxy have you depopulated' counter seems entertaining.

**Comment by McloughlinGuy** [comment]
> Then I found out all the pirates are 17 year old kids and idly wondered how many teenagers I'd murdered.

Not necessarily all of them, but yeah a good number of them are. Also I'm pretty sure combat XP is calculated based off of the price of each ship you've disabled, you don't actually have to kill anyone to gain XP though you most likely will have killed a lot of people to get a lot of at.

**Comment by SimulatedKnave** [comment]
Hmmm. I think you're right. Probably makes the above impractical, then.

**Comment by McloughlinGuy** [comment]
> Hmmm. I think you're right. Probably makes the above impractical, then.

Not necessarily, you could just show how many ships you've killed from each faction which would still be an interesting metric to see, plus you could calculate how many crew you killed from there yourself.

**Comment by ThrawnCA** [comment]
> plus you could calculate how many crew you killed from there yourself.

Not if you only disabled them.

Of course, from an ethical perspective, that's probably a kill.
